src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -213,7 +213,7 @@ package com.videojs.providers{
         }
         
         public function get playing():Boolean{
-            return _isPlaying;
+            return _playbackStarted;
```

Comment:
yes, from what @bclwhitaker and I understood, this is what this variable actually means. It's true from when a user as pressed play/seeked and until the video has ended.